scripts/install_plugin_modules.js

```diff
@@ -120,7 +120,7 @@ function assertWorkspace () {
 }
 
 function install () {
-  exec('yarn', { cwd: folder() })
+  exec('yarn --ignore-engines', { cwd: folder() })
```

Comment:
Couldn't this theoretically cause issues?

Reply:
`npm` operates this way by default (albeit with warnings outputted). If we don't do this, we can't actually test on Node 8, which people may be using with `@google-cloud/pubsub`.

Reply:
Is that because it doesn't support Node 8? If that's the case, I don't think it should be tested anyway.

Reply:
The dropped support is in a version of `semver` that's a dependency of `@google-cloud/pubsub` (transitively via `google-gax`).